modules/event-settings-wheel-duration-picker/src/TimePickerModule.tsx

```diff
@@ -0,0 +1,33 @@
+import { requireNativeViewManager } from "expo-modules-core"
+import React from "react"
+import { StyleProp, ViewProps, ViewStyle } from "react-native"
+import { NativeViewGestureHandler } from "react-native-gesture-handler"
+
+type OnDurationChangeEvent = {
+  duration: number
+}
+
+export type TimePickerProps = {
+  initialDuration: number
+  onDurationChange: (event: { nativeEvent: OnDurationChangeEvent }) => void
+  style?: StyleProp<ViewStyle>
+} & ViewProps
+
+const NativeView: React.ComponentType<TimePickerProps> =
+  requireNativeViewManager("TimePicker")
+
+export const TimePickerView = ({
+  style,
+  onDurationChange,
+  ...props
+}: TimePickerProps) => {
+  return (
+    <NativeViewGestureHandler disallowInterruption>
+      <NativeView
+        style={style}
+        onDurationChange={onDurationChange}
+        {...props}
+      />
+    </NativeViewGestureHandler>
+  )
+}
```

Comment:
On iOS we don't want to use `requireNativeViewManager` because we'll just use the built-in duration picker component instead.